data.sql

```diff
@@ -3,4 +3,14 @@ INSERT INTO
 	VALUES	(1, 'Agumon', '2020-02-03', 0, true, 10.23),
 			(2, 'Gabumon', '2018-11-15', 2, true, 8),
 			(3, 'Pikachu', '2021-01-07', 1, false, 15.04),
-			(4, 'Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+			(4, 'Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO 
+	animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES	(5, 'Charmander', '2020-02-08', 0, false, -11),
+			(6, 'Plantmon', '2021-11-15', 2, true, -5.7),
+			(7, 'Squirtle', '1993-04-02', 3, false, -12.13),
+			(8, 'Angemon', '2005-06-12', 1, true, -45),
+			(9, 'Boarmon', '2005-06-05', 7, true, 20.4),
```

Comment:
* Kindly correct the DOB of animal `Boarmon` is should be `born on Jun 7th, 2005` [Check requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=born%20on%20Jun%207th%2C%202005)

Reply:
Thanks for the information. I will update it